tests/intel_fpga_test.sh

```diff
@@ -118,6 +118,10 @@ run_all() {
 
     #Unique nested sdfg
     run_sample fpga/unique_nested_sdfg_fpga two_vecAdd "\n"
+
+    ## BLAS
+    run_sample blas/nodes/axpy_test blas_axpy_test "\n" --target intel_fpga
```

Comment:
This should not be necessary, the test can know from the DaCe config which vendor is currently being run

Reply:
How is this configured whether to run on CPU or on FPGA, which value in the dace.conf determines CPU or FPGA?

Reply:
Right, you would still need `--target fpga`, but you can query the vendor with:
```
>>> dace.config.Config.get("compiler", "fpga_vendor")
'intel_fpga'
```